### ISSUE TITLE
control-group: improve JAWS readability

### ISSUE THREAD

**Issue by aristov** [bem/bem-components#1522]
JAWS should read 
- [x] `control-group` as "group start" / "group end"
- [x] `content` field as list of specified controls

Check and fix divergencies.

**Comment by aristov** [comment]
http://www.w3.org/TR/wai-aria/roles#group

**Comment by sipayRT** [comment]
@aristov 
For using `role=group` attribute we need to add also `aria-label`. But this attribute should be filled. For example:

```
<div role="group" aria-label="group">
    <div>option 1</div>
    <div>option 2</div>
</div>
```

But in this case JAWS text will be as "group start group" instead of "group start". We can change role to `list`, but it's not right way. 

@ruslansagitov maybe, you can help us? For clarify:
we have group of buttons with wrapper called as `control-group`. What role should we use for this wrapper and for child elements?

**Comment by sipayRT** [comment]
also child elements can be not just a button but items with custom tags too.

**Comment by aristov** [comment]
@sipayRT we can use `aria-label="."` as the simplest solution. It provides the desired behaviour ("group start" / "group end") and doesn't add redundant things. 
But I propose to raise the question about addition something like `title` field to `control-group` and `checkbox-group` blocks API. It can be expressed as `title` or `aria-label` attribute without visual representation in `islands` theme, as well as used to make a visible title in some other custom theme.
// cc @narqo, @dfilatov

**Comment by ruslansagitov** [comment]
Well, I’m not quite sure of what the `control-group` is but you can use the group role without the `aria-label` property. It will be said as “group start … group end”. I guess, it’s enough for `control-group`.

Also, you need to allow users to set the `aria-label` manually if they really need it.

**Comment by sipayRT** [comment]
@ruslansagitov Tested in FF with JAWS(version 16). With HTML like 

```
<div role="group">
    <div> option 1</div>
    <div> option 2</div>
</div>
```

it said just "option one, option two". After adding `aria-label` it said properly

**Comment by ruslansagitov** [comment]
@sipayRT Try to use the Tab key. I guess, inside the `control-group`, you place a control, which steals focus on Tab. JAWS should say that now you are on some control, which is inside some group. I can’t confirm it right now but it should work that way.

I think you should not do anything than adding `role=group` in this case. Try to test in other screen readers, in NVDA, for example.
